### ISSUE TITLE
0.10.2 fails to create nomad job

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-nomad#10]
_This issue was originally opened by @michaelw as hashicorp/terraform#16011. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
Terraform v0.10.2 (Darwin)
Nomad v0.6.0 (Linux)

### Terraform Configuration Files
```hcl
provider "nomad" {
  version = "~> 0.1"
  address = "http://nomad.service.consul:4646"
}

resource "nomad_job" "jenkins" {
  jobspec = "${file("${path.module}/example.nomad")}"
}

terraform {
  backend "consul" {
    address = "consul.service.consul:8500"
    path    = "test/terraform_state"
  }
}
```

### Debug Output
https://gist.github.com/michaelw/67a147bb83de73724830d0d0c6307b46

### Expected Behavior
nomad job should have been submitted.

### Actual Behavior
TF failed to submit nomad job.

### Steps to Reproduce
0. `nomad init`
1. `terraform init`
2. `terraform apply`

**Comment by tombuildsstuff** [comment]
Hey @michaelw

Thanks for opening this issue. Taking a quick look into this - I believe this should be fixed by #7

Thanks!

**Comment by radeksimko** [comment]
#7 was just merged and will be part of the next release.
